controls/V-72067.rb

```diff
@@ -146,17 +124,17 @@
 
 boot=UUID=05c000f1-a213-759e-c7a2-f11b7424c797
 
-Reboot the system for the changes to take effect."
+Reboot the system for the changes to take effect.
+"
+  tag "fix_id": "F-78419r2_fix"
 
   describe package('dracut-fips') do
-    it { should be_installed}
+    it { should be_installed }
+  end
+  describe grub_conf(GRUB_CONF_PATH, 'default') do
```

Comment:
This should probably use `grubby` to get the actual configuration strings instead of relying on internal parsing of potentially incorrect files.